docs/source/daml/reference/choices.rst

```diff
@@ -72,10 +72,55 @@ Controllers
 
   The conjunction of **all** the parties are required to authorize when this choice is exercised.
 
-.. _daml-ref-anytime:
+.. _daml-ref-consumability:
+
+Consumability
+=============
+
+If no qualifier is present, choices are *preconsuming*. This behavior is also usually referred to simply as *consuming*.
```

Comment:
Once we have merged https://github.com/digital-asset/daml/pull/4164, this will be wrong. No qualified means `consuming` which is like `preconsuming` + the additional privacy leak.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4134/commits/bf8c3d9a27f12feadb1499a45e53a6cacd0454ef